app/src/main/java/app/gamenative/ui/model/MainViewModel.kt

```
@@ -652,6 +709,34 @@ class MainViewModel @Inject constructor(
+                } else if (!hasSentPlayingNotification) {
+                    val windowBase = window.className.substringAfterLast('\\')
+                        .substringAfterLast('/').lowercase().removeSuffix(".exe")
+                    if (windowBase in CORE_WINE_PROCESSES) return@let
+
+                    val launchExeNames = SteamService.getWindowsLaunchInfos(gameId)
+                        .map { Paths.get(it.executable.replace('\\', '/')).name.lowercase() }
+                    if (isLaunchExeRunning(launchExeNames)) {
+                        val shouldLaunchRealSteam = try {
+                            val container = ContainerUtils.getContainer(context, appId)
+                            container.isLaunchRealSteam()
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)